frontend/src/lib/k8s/cluster.ts

```diff
@@ -55,7 +40,13 @@ export interface KubeObjectInterface {
   kind: string;
   apiVersion?: string;
   metadata: KubeMetadata;
-  [otherProps: string]: any;
+  spec?: any;
+  status?: any;
+  items?: any[];
+  actionType?: any;
+  lastTimestamp?: string;
+  key?: any;
+  // [otherProps: string]: any;
```

Comment:
Why this comment? This is also likely to make ts complain.
The idea here is that instead of having to cover always what k8s API may have, we allow for other unknown props to be defined.